### PR TITLE
Add my extensions (Hashing and Tab Info)

### PR DESCRIPTION
**Pull request by techguy16**
Hashing:
* Hash SHA1, SHA256 and MD5 strings

Tab Info:
* Get info about the current tab, i.e height.

**Both extensions need to be run unsandboxed**

### CODE REVIEW

**Comment by SharkPool-SP** [comment]
These blocks can be merged into other existing extensions.

File: extensions/techguy16/hashing.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            await loadScript('https://web.archive.org/web/20151121160242if_/https://crypto-js.googlecode.com/svn/tags/3.1.2/build/rollups/sha256.js?_=1448121771646');
+        }
+
+        const hash = CryptoJS.SHA256(input).toString();
+        return hash;
+    }
+
+    async function md5Hash(input) {
+        if (typeof md5 !== 'function') {
+            await loadScript('https://web.archive.org/web/20151121123050if_/http://crypto-js.googlecode.com/svn/tags/3.1.2/build/rollups/md5.js');
```
Comment (GarboMuffin) [inline extensions/techguy16/hashing.js:32]: no you cant do that, extensions must be self contained

File: extensions/techguy16/hashing.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        blocks: [
+            ['R', 'SHA-1 of %s', 'sha1Hash', 'Hello, world!'],
+            ['R', 'SHA-256 of %s', 'sha256Hash', 'Hello, world!'],
+            ['R', 'MD5 of %s', 'md5Hash', 'Hello, world!']
+        ],
+        menus: {},
+        url: 'https://github.com/techguy16'
+    };
+
+    ScratchExtensions.register('Hashing', descriptor, {
```
Comment (GarboMuffin) [inline extensions/techguy16/hashing.js:49]: scratchx is deprecated. dont use it

**Comment by techguy16** [comment]
You can fix the problems.

**Comment by NexusKitten** [comment]
> Can you fix the problems?

Before anyone can provide any help, please **_fully_** read through the [Extension Documentation](https://docs.turbowarp.org/development/extensions/introduction) and [CONTRIBUTING.md](https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md). A lot of the issues in your extensions should've been covered there.

**Review by NexusKitten — CHANGES_REQUESTED** [review]
https://github.com/TurboWarp/extensions/pull/1032#issuecomment-1718392741

**Comment by techguy16** [comment]
Well I give up. I do give permission to use my code in the future if you want to.

**Pull request closed without merging**